front/public/locales/de/stdcm.json

```
@@ -1,4 +1,8 @@
 {
+  "noConfigurationFound": {
+    "text": "Bitte wenden Sie sich an das Admin Team, damit Sie ihm weiterhelfen können.",
```

As for the actual meaning of the translations, it looks nice to me, even tho I'm not fluent german. 
Our fellow @Tristramg might not be available before next week to validate it however.

Edit, if we want to keep the play on word:
```suggestion
    "text": "Bitte wenden Sie sich an das Admin Team, damit sie Sie wieder auf die richtige Spur bringen können.",
```

---

Didn't even notice this play on word haha. This is great.

---

Hi all, this has been a busy month for me: I'm trying to break a housing contract, moving to a new apartment, and leaving my current job. I'll sift through all the comments in July. Thanks for getting back to me and talk soon :)